src/test/sql/sql_translator_test.cpp

```diff
@@ -461,6 +461,21 @@ TEST_F(SQLTranslatorTest, SelectListAliasesDifferentForSimilarAggregatesInSubque
   EXPECT_LQP_EQ(actual_lqp, expected_lqp);
 }
 
+TEST_F(SQLTranslatorTest, RepeatingAggregates) {
+  // See #1902. Optimally, these queries would produce correct results. Until then, at least make sure they don't
+  // produce any wrong ones.
+
+  EXPECT_THROW(compile_query("SELECT COUNT(*) FROM (SELECT COUNT(*) FROM t WHERE a <= 1234) t2"),
+               InvalidInputException);
+
+  EXPECT_THROW(compile_query("SELECT COUNT(a) FROM (SELECT a, COUNT(a) FROM t GROUP BY a) t2"), InvalidInputException);
+
+  EXPECT_THROW(compile_query("SELECT COUNT(a) FROM (SELECT a, COUNT(a) AS b FROM t GROUP BY a) t2"),
+               InvalidInputException);
+
+  EXPECT_THROW(compile_query("SELECT AVG(a) FROM (SELECT a, AVG(a) FROM t GROUP BY a) t3"), InvalidInputException);
```

Comment:
I would omit the table aliases `t2`, `t3` as they do not contribute to the problem.

Reply:
Subqueries in the FROM part without an alias are illegal.

Reply:
Oh, I was not aware of that. Thank you!